src/fmv.cpp

```diff
@@ -25,18 +25,53 @@ static f32 Percent(f32 max, f32 percent)
     return (max / 100.0f) * percent;
 }
 
-static void RenderSubtitles(Renderer& rend, const char* msg, float x, float y, float w, float h)
+static void RenderSubtitles(Renderer& rend, std::vector<const SubTitle*> msg, float x, float y, float w, float h)
 {
+
+	const char* prev_sub = "BUG";
```

Comment:
should probably default to "" instead of BUG - might look bad if that ever randomly appears